.gitignore

```diff
@@ -1,6 +1,6 @@
 # tools/jenkins-status:
 node_modules/
-package-lock.json
+/package-lock.json
```

Comment:
I don't think this entry is necessary. Since there's no package.json in /, there won't be a package-lock.json either. I think you can just remove it entirely. The only `package.json` file is in `tools/jenkins-status`.

Reply:
```suggestion
```

Reply:
Done